suites/rbd/qemu/workloads/qemu_bonnie.yaml

```diff
@@ -1,5 +1,5 @@
 tasks:
 - qemu:
     all:
-      test: https://ceph.com/git/?p=ceph.git;a=blob_plain;f=qa/workunits/suites/bonnie.sh
+      test: https://git.ceph.com/?p=ceph.git;a=blob_plain;f=qa/workunits/suites/bonnie.sh
```

Comment:
FYI, I have found http**s**://git.ceph.com to not work...

See: http://tracker.ceph.com/issues/11406